### DISCUSSION TITLE
Roadmap 2023

### DISCUSSION THREAD

**Discussion by marcgc21** [ruxailab/RUXAILAB#90] [Ideas]
Hola a todos! Estamos terminando el año y nos gustaría poder empezar a planear y crear diferentes ideas para poder integrar en la plataforma. Estaría bien podder tener una lista para poder trabajar en ella individualmente, que os parece?

Propondría los siguientes ítems:

- Screen capture and screen recorder
- Voice recorder and audio editor
- Collaborative feedback tool

**Comment by KarinePistili** [comment]
Hello! I'd like to revive this discussion, so we can talk a little bit more about the "Collaborative feedback tool". 

The screen capture and recorder is being developed now for User Tests, but we still don't have something that allows a collaborative feedback, for groups answers or analysis.

If anyone has any suggestions, please share with us :)
